test/httpPipelineLogLevelTests.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { HttpPipelineLogLevel, httpPipelineLogLevelToString } from "../lib/httpPipelineLogLevel";
+import * as assert from "assert";
+
+describe("httpPipelineLogLevelToString()", () => {
+    it(`should work with ERROR`, () => {
+        assert.strictEqual(httpPipelineLogLevelToString(HttpPipelineLogLevel.ERROR), "ERROR");
```

Comment:
Might be worth considering [string enums](https://stackoverflow.com/questions/15490560/create-an-enum-with-string-values-in-typescript#15491832) (the enum value implicitly converts to string) instead of having a conversion method

Reply:
I didn't know that typescript added string enums. I'll change HttpMethod and HttpPipelineLogLevel to use those instead.